UserAdminBundle/Form/Type/UserType.php

```diff
@@ -88,24 +108,16 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     'invalid_message' => 'fos_user.password.mismatch',
                     'group_id' => 'authentication',
                     'sub_group_id' => 'identifier',
+                    'required' => false,
+                ))
+                ->add('editAllowed', 'radio', array(
+                    'label' => 'open_orchestra_user_admin.form.user.edit_allowed',
+                    'required' => false,
+                    'group_id' => 'information',
+                    'sub_group_id' => 'profil',
                 ));
-        }
-        $builder
-            ->add('language', 'choice', array(
-                'choices' => $this->getLanguages(),
-                'label' => 'open_orchestra_user_admin.form.user.language',
-                'group_id' => 'preference',
-                'sub_group_id' => 'backoffice',
-            ))
-            ->add('languageBySites', 'oo_language_by_sites', array(
-                'label' => false,
-                'sites_id' => $sitesId,
-                'group_id' => 'preference',
-                'sub_group_id' => 'language',
-            ));
-
-        if ($options['edit_groups']) {
-            $builder->addEventSubscriber(new UserGroupsSubscriber());
+            $builder->addEventSubscriber(new UserProfilSubscriber($options['current_user'], $this->objectManager));
```

Comment:
why use options current_user if only subscribers need it and not get current user inside subscriber and use dependency injection